api/swagger.yaml

```diff
@@ -6649,6 +6662,110 @@ definitions:
     additionalProperties:
       type: "string"
 
+  ImageManifestSummary:
+    description: |
+      ImageManifestSummary represents a summary of an image manifest.
```

Comment:
I think we can either rename the definition here, or add a `x-go-name: "ManifestSummary"`

Reply:
Went with the `x-go-name`